README.md

```diff
@@ -1,2 +1,16 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment  
+
+## Introduction of Myself
+
+Hi everyone!  
+I'm **Xiaoran Cheng** from China. You can call me **David**.
+
+1. *Hometown*: Beijing
+2. *Interest*:
+   * Soccer
+   * Reading
+
+You guys can follow me on [Facebook](https://www.facebook.com/profile.php?id=100005247065834)
+
+![This is me~](https://scontent.xx.fbcdn.net/v/t1.0-9/15977342_645541108964119_622605024512763586_n.jpg?oh=f7f0f31961a4ed3294bd152ea965b97e&oe=58D9D2A1)
```

Comment:
Good use of picture! I'm not sure you need the ~ at the end of the caption, though.